CryptoBinPkg/CryptoBinPkg.dec

```diff
@@ -1,30 +1,30 @@
-## @file
-#  Cryptographic Library Package for UEFI Security Implementation.
-#  PEIM, DXE Driver, and SMM Driver with all crypto services enabled.
-#
-#  Copyright (c) 2009 - 2021, Intel Corporation. All rights reserved.<BR>
-#  Copyright (c) 2020, Hewlett Packard Enterprise Development LP. All rights reserved.<BR>
-#  SPDX-License-Identifier: BSD-2-Clause-Patent
-#
-##
-
-################################################################################
-#
-# Defines Section - statements that will be processed to create a Makefile.
-#
-################################################################################
-[Defines]
-  DEC_SPECIFICATION              = 0x00010005
-  PACKAGE_NAME                   = CryptoBinPkg
-  PACKAGE_GUID                   = 19189174-7994-4ccf-b5bf-eeb43d690d8d
-  PACKAGE_VERSION                = 1.0
-
-[Includes]
-
-[LibraryClasses]
-
-[Protocols]
-
-[PcdsFixedAtBuild]
-
-[Guids]
+## @file

```

Comment:
What is the size difference in the binaries with the addition of Pkcs7Encrypt ?

If the size difference is large enough, maybe this will need to be marked as a breaking change?